inference-engine/tests/unit/vpu/base/graph_transformer_tests.cpp

```diff
@@ -179,6 +201,41 @@ Stage TestModel::addStage(
 
     return stage;
 }
+
+Stage TestModel::_addStage(
```

Comment:
why do you need this modification of `addStage`? You cannot use `addStage` function? Anyway, even if it is needed, it is better to reuse code of these functions instead of duplicate it (they are almost the same apart from one line)